Tutorials/Cache-Static-Assets/index.md

```diff
@@ -0,0 +1,99 @@
+---
+versionFrom: 9.3.0
+---
+
+# Cache static assets
+
+Example to cache static assets by file extension, but excluding Umbraco BackOffice assets.
+
+```csharp
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Microsoft.Net.Http.Headers;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Umbraco.Cms.Core.Composing;
+using Umbraco.Cms.Core.Configuration.Models;
+using Umbraco.Cms.Core.DependencyInjection;
+using Umbraco.Cms.Core.Hosting;
+using Umbraco.Extensions;
```

Comment:
For V10 there is an ambiguous reference between different `IHostingEnvironment` implementations; best make it explicit:

```suggestion
using Umbraco.Extensions;
using IHostingEnvironment = Umbraco.Cms.Core.Hosting.IHostingEnvironment;
```

Reply:
Seems to work fine to me without this change, which reference `Umbraco.Cms.Core.Hosting.Umbraco.Cms.Core.Hosting` , but yes if one include `Microsoft.AspNetCore.Hosting` or maybe in global usings is may be ambiguous.